src/organisms/Player.js

```diff
@@ -0,0 +1,43 @@
+import { Animal } from './Animal';
+import { getDirection } from '../utilities/getDirection';
+
+export class Player extends Animal {
+  constructor(positionY, positionX, age = 0) {
+    super(5, 4, positionY, positionX, age);
+  }
+
+  action(board) {
+    return new Promise((resolve) => {
+      const handleKeyPress = (event) => {
+        const direction = getDirection(event.key);
```

Comment:
The `getDirection` function could be a method of the `Player` class, it's not used anywhere else